src/components/activities/ActivityCard.jsx

```diff
@@ -140,18 +143,23 @@ class ActivityCard extends Component {
     );
   }
 
-  renderCheckbox = () => (
-    (
-      <input
-        type='checkbox'
-        name='checkbox'
-        value={this.props.id}
-        className='activity__checkbox'
-        checked={this.state.isActivityChecked}
-        onChange={this.handleActivityChecked}
-      />
-    )
-  );
+  renderCheckbox = () => {
+    if (this.props.showCheckBox) {
```

Comment:
I see what you are doing here but remember what I suggested last time about evaluating this before the return. This is a classic example of improving readability so we pass the right `jsx` into the return statement. Let me know if that makes sense.

Reply:
In truth, this was a quick fix as this is a bug, I feel this will later need to be refactored but I will address your feedback and change it.